tests/server/test_server_validation.py

```diff
@@ -21,6 +22,29 @@ def test_with_validator(both_fake_remote_clients):
     assert validator.valid
 
 
+def test_with_validator_json_response(both_fake_remote_clients, capsys):
+    """ Test that the validator writes compliant JSON when requested. """
+    from optimade.server.main_index import app
+
+    validator = ImplementationValidator(
+        client=both_fake_remote_clients,
+        index=both_fake_remote_clients.app == app,
+        respond_json=True,
+    )
+    try:
+        validator.validate_implementation()
+    except Exception:
+        print_exc()
```

Comment:
Don't you want the test to fail if this happens? It should be fine if you just don't do the `try/except`?

Reply:
I probably do, yes, I was just following the template from the test above. I think this is leftover from before I introduced "InternalError" and we handled validator failures differently, thanks for spotting it.